Examples/MAX32665/SPI_3Wire/project.mk

```diff
@@ -0,0 +1,15 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://analog-devices-msdk.github.io/msdk/USERGUIDE/#build-system
+
+# **********************************************************
+
+# Add your config here!
```

Comment:
If project is not supported on the EVKIT, add the following lines:

```Makefile
BOARD = FTHR

ifeq ($(BOARD),EvKit_V1)
$(error ERR_NOTSUPPORTED: This project is not supported for the MAX32665EVKIT)
endif
```

Then, re-run [MSDKGen](https://github.com/Analog-Devices-MSDK/MSDKGen).  It will detect that you have set a different default `BOARD` and update the project settings to match.

Reply:
The example is supported on both FTHR and FTHR2. So, I did not add the line "BOARD = FTHR". I added the error throwing when the board is chosen as EVKIT. I also tried this; when I try to build the code for EvKit, I get the error.
Thanks.